src/defaults.js

```diff
@@ -1,4 +1,7 @@
 export default {
   maxQueriesPerSecond: 50,
-  minQueriesPerSecond: 1
+  minQueriesPerSecond: 1,
+  defaultQueriesPerSecond: 35,
```

Comment:
Yes, adding a hint, that you can't set the QPS to 50 would be good here:

https://github.com/ubilabs/node-geobatch/pull/12/files/6d35c524cd74a4df282f22e734f5b34309e35d28#diff-04c6e90faac2675aa89e2176d2eec7d8L166